content/20.treatments.md

```diff
@@ -185,16 +185,15 @@ Entry of SARS-CoV-2 into the cell depends on the ACE2 receptor and the enzyme en
 In principle, drugs that reduce the expression of these proteins or sterically hinder viral interactions with them might reduce viral entry into cells.
 
 **Current Evidence.**
-Angiotensin-converting enzyme inhibitors and angiotensin II receptor blockers (ARBs) are among the most commonly prescribed medications [@url:https://clincalc.com/DrugStats/Drugs/Lisinopril; @doi:10.1056/NEJMp1901657].
+Angiotensin-converting enzyme (ACE) inhibitors and angiotensin II receptor blockers (ARBs) are among the most commonly prescribed medications [@url:https://clincalc.com/DrugStats/Drugs/Lisinopril; @doi:10.1056/NEJMp1901657].
 In the United States, for example, they are prescribed well over 100,000,000 times annually.
 Data from some animal models suggest several but not all ACE inhibitors and several ARBs increase ACE2 expression in cells of some organs [@doi:10.1093/cvr/cvaa097].
 Clinical studies have not established whether plasma ACE2 expression is increased in humans treated with these medications [@doi:10.1093/europace/euw246].
 While randomized clinical trials are ongoing, a variety of observational studies have examined the relationship between exposure to ACE inhibitors or ARBs and outcomes in patients with COVID-19.
-In an observational analysis of deaths among patients hospitalized with COVID-19, Mehra and colleagues reported that treatment with an ACE inhibitor at time of admission corresponded to an odds ratio of 0.33 [@doi:10.1056/NEJMoa2007621].
-In contrast, the authors did not find evidence of an association between exposure to an ARB at time of hospitalization and death.
-Because observational studies are subject to confounding, randomized controlled trials are the standard means of assessing the effects of medications, and the findings of this study cannot be interpreted as indicating a protective effect of the drug.
+An observational study of the association of ACE inhibitor or ARB exposure on outcomes in COVID-19 was retracted from the _New England Journal of Medicine_. [@doi:10.1056/NEJMoa2007621].
+Moreover, because observational studies are subject to confounding, randomized controlled trials are the standard means of assessing the effects of medications, and the findings of the various observational studies bearing on this topic cannot be interpreted as indicating a protective effect of the drug [@doi:10.1161/CIRCRESAHA.120.317205; doi:10.2215/CJN.03530320].
```

Comment:
```suggestion
Moreover, because observational studies are subject to confounding, randomized controlled trials are the standard means of assessing the effects of medications, and the findings of the various observational studies bearing on this topic cannot be interpreted as indicating a protective effect of the drug [@doi:10.1161/CIRCRESAHA.120.317205; @doi:10.2215/CJN.03530320].
```